docs/source/Plugin/P144.rst

```diff
@@ -0,0 +1,68 @@
+.. include:: ../Plugin/_plugin_substitutions_p14x.repl
+.. _P144_page:
+
+|P144_typename|
+==================================================
+
+|P144_shortinfo|
+
+Plugin details
+--------------
+
+Type: |P144_type|
+
+Name: |P144_name|
+
+Status: |P144_status|
+
+GitHub: |P144_github|_
+
+Maintainer: |P144_maintainer|
+
+Used libraries: |P144_usedlibraries|
+
+Description
+-----------
+
+This plugin supports the dust sensor used in the Ikea Vindriktning. The PM1006K sensor uses a serial protocol to communicate with the controller. The Vindriktning contains a controller that sets up the sensor and reads the value regularly. 
+This plugin is intended to eavedrop this communication. All sensor reading results are copied to the plugin.
+
+Note: The current plugin does not support direct communication with the PM1006(K) sensor. However, the plugin can be extended alter to also support standalone control of this device.
+
+Hardware
+---------
+
+The Vindrikting can be bought by Ikea. The device can be opened easily by 4 screws. Inside there is room for a Wemos D1 mini or a similar form factor. The 5 Volt power is available on the Vindrikning solder pads. 
+Also the data received from the sensor is available on a solder pad. Images of a hacked Vindrikning are available on the internet.
+
+Connections
+^^^^^^^^^^^^
+
+.. image:: P144_Connection.jpg
+   :alt: Connecting the Vindrikning
+
+
+Device configuration
+--------------------
+
+.. image:: P144_DeviceConfiguration.png
+   :alt: Device configuration
+
+**Name** : A unique name should be entered here.
+
+**Enabled** : The device can be disabled or enabled. When not enabled the device should not use any resources.
+
+**Serial Port** : Select the serial port to be used. Choices depend on the type of ESP chip being used. See serial port in this manual for the available options.
```

Comment:
Other plugins using ESPEasySerial have a common reference/link to that documentation, you could borrow/copy that here too.